### DISCUSSION TITLE
Heliocentric

### DISCUSSION THREAD

**Discussion by Osphie-Mia** [Stellarium/stellarium#4014] [Q&A] [answered]
Hi,

I'm strating on stellarium, i have few question on it.

Do exist a rescearch fonction on fonctinnality of stellarium ? It can scearch a planet, but can we scearch a fonction of the softwear ?

And the principal question : is on the position of the point of view. Can deplace the point of view form earth to the sun, to have an heliocentric observation of the sky ? To pass of geocentric to heliocentric.

Thank you
Osphie-Mia

**Comment by gzotti** [comment, marked as answer]
Please read the introductory chapters of the User Guide to get a rapid introduction to the program.
